src/components/Comments.js

```diff
@@ -241,14 +263,15 @@ class Comments extends React.Component<void, CommentsProps, CommentsState> {
   renderCommentLink() {
     const { commentList } = this.state;
     const commentCount = commentList && commentList.length > 0 ? commentList.length : 0;
-    const commentLabel = commentCount > 0 ? `Comment (${commentCount})` : 'Comment';
+    const commentLabelFormat = `Add a comment | 1 Comment | ${commentCount} Comments`;
```

Comment:
This should be a plain string like:
```
const commentLabelFormat = 'Add a Comment|1 Comment|{} Comments`;
```
The {} will automatically get replaced with the count....(I also removed the spaces around the "|" character... and changed the C in comment to be capitalized everywhere to make it consistent with the existing "Show 360° View" link).